lib/users.js

```diff
@@ -245,6 +245,36 @@ exports.register = function (server, options, next) {
         }
       }
     }
+  }, {
+    method: 'GET',
+    path: options.basePath + '/lms/user',
+    handler: handlers.actHandlerNeedsUser('get_lms_link'),
+    config: {
+      description: 'Get LMS connecting URL',
+      tags: ['api', 'lms'],
+      plugins: {
+        'hapi-swagger': {
+          responseMessages: [
+            {code: 200, message: 'OK'}
+          ]
+        }
+      }
+    }
+  }, {
+    method: 'POST',
+    path: options.basePath + '/lms/badge',
+    handler: actHandlerCheckCertificate,
```

Comment:
Not really descriptive of what it actually does. Perhaps should be specific, or take an argument for the act executed after checking the cert.